autotst/job/job.py

```diff
@@ -397,8 +488,9 @@ def calculate_species(self, species):
                 for running in currently_running:
                     if not running.is_alive():
                         currently_running.remove(name)
-                time.sleep(15)
```

Comment:
Is this indentation level change intentional?
Looks a bit like the while loop will now run indefinitely without delay.

Reply:
Good catch. That was an intentional change in indentation but we should have a `time.sleep` in this loop. The latest commit reflects that change.